src/components/sections/AboutSection.tsx

```diff
@@ -0,0 +1,85 @@
+import Socha from "../../../public/Stani2.png"
+import InfoCard from "../InfoCard"
+
+const AboutSection = () => {
+  return (
+    <div className="mt-10 flex justify-center items-center">
+      <div className="grid grid-cols-2 grid-rows-6">
+        <div className="row-start-1 row-end-4 col-start-1 col-end-2">
+          <img src={Socha} alt="" />
```

Comment:
when using the public path, we can put it directly to the `src="/Stani2.png"`
- also, provide the alt value